src/tracks/gosling-track/gosling-track.ts

```diff
@@ -1472,6 +1501,31 @@ const factory: PluginTrackFactory<Tile, GoslingTrackOptions> = (HGC, context, op
                 }
             });
         }
+
+        /**
+         * Used in drawTile()
+         * Checks if the track has marks which are stretchable. Stretching
+         * is not supported for circular layouts or 2D tracks
+         */
+        #hasStretchableGraphics() {
+            const stretchableMarks = ['bar', 'line', 'rect', 'area'];
+            return (
+                this.options.spec.experimental?.stretchGraphics ||
+                (!Is2DTrack(this.getResolvedTracks()[0]) &&
+                    this.options.spec.layout !== 'circular' &&
+                    stretchableMarks.includes(this.options.spec.mark || ''))
```

Comment:
Could we still enable testing the original rendering with `stretchGraphics: false`?

```suggestion
                (typeof this.options.spec.experimental?.stretchGraphics  == 'undefined' &&
                    !Is2DTrack(this.getResolvedTracks()[0]) &&
                    this.options.spec.layout !== 'circular' &&
                    stretchableMarks.includes(this.options.spec.mark || ''))
```